apps/web/components/Navbar.tsx

```diff
@@ -1,32 +1,82 @@
 "use client";
 
+// Navbar.tsx
+
 import Link from "next/link";
 import EventSeaLogo from "../public/icons/EventSeaLogo";
 import WalletIcon from "../public/icons/WalletIcon";
-import CreateEvent from "@/components/create-event-form"
-
+import CreateEvent from "@/components/create-event-form";
 import { Button } from "./ui/Button";
 import { SearchBar } from "./SearchBar";
+import { useState } from "react";
+import { useSDK, MetaMaskProvider } from "@metamask/sdk-react";
+import { formatAddress } from "./../lib/utils";
+
+export const ConnectWalletButton = () => {
+  const [account, setAccount] = useState<string | undefined>();
+  const { sdk, connected, connecting } = useSDK();
+
+  const connect = async () => {
+    try {
+      const metaMaskAccount = (await sdk?.connect()) as string;
```

Comment:
is this type casting to `string` correct ?
isn't `await sdk?.connect()` return an array of accounts ?

Reply:
yeah it does but like one wallet can have multiple accounts I believe.
@chin-flags

Reply:
oh okay. I got it now. Thank you so muchh. @chin-flags